core/src/defaultconfigs/bareos-sd.d/device/FileStorage.conf.in

```diff
@@ -1,4 +1,5 @@
 Device {
+  Description = "Multiplied file device. A connecting Director must have the same Name and MediaType."
   Name = FileStorage
```

Comment:
What about placing the number just below the name, so people will better understand why they got FileStorage0001 etc. Isn't 10 a too high number?

More you have read/write device, more volumes you will use, and you will have to explain users, that they will need as much volume per day equal the number of devices if concurrency is high enough of course.

```suggestion
  Name = FileStorage
  Count = 10                          # Number of instances
```

Maybe going to 4 will be easier to explain and document like you have 2 writer / 2 reader for AI etc...